webapp/channels/src/components/post_view/reaction_list/reaction_list.tsx

```diff
@@ -151,7 +153,10 @@ export default class ReactionList extends React.PureComponent<Props, State> {
         }
 
         let emojiPicker = null;
-        if (this.props.canAddReactions && this.state.emojiNames?.length < EmojiConstant.MAX_EMOJI_NAME_LENGTH) {
+        const getState = store.getState;
+        const state = getState() as GlobalState;
+        const config = getConfig(state);
+        if (this.props.canAddReactions && this.state.emojiNames?.length < Number(config.UniqueEmojiReactionLimitPerPost)) {
```

Comment:
Could you use `parseInt(config.UniqueEmojiReactionLimitPerPost, 10)` instead of `Number` to convert this to a number? That's the more standard way of converting a string to a number